spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/integration/IntegrationAutoConfigurationTests.java

```diff
@@ -144,12 +193,12 @@ public void load(String... environment) {
 		load(null, environment);
 	}
 
-	private void load(Class<?> config, String... environment) {
+	private void load(Collection<Class<?>> configs, String... environment) {
```

Comment:
I tend to use a `Class<?>[]` for this. This prevents the translation from list and is as easier to read.